portfolio/src/main/webapp/about.html

```diff
@@ -64,39 +65,44 @@ <h3 style = "background-color: hsl(0, 0%, 100%, 0.5); padding:5px">Coding Langua
                         </div>
 
                         <div class="title">
-                            <h3 style = "font-size: 25px">Things That Make Me Happy</h3>
+                            <h3 style = "font-size: 32px">Things That Make Me Happy</h3>
                         </div>
 
                         <div class="sect-4">
-                          <p>My Dogs.</p>
+                          <p style = "font-size: 14pt">My Dogs.</p>
                           <img src="/images/dogs.png" alt = "Picture of my Dogs."/>
+                          <p> Yes, there are three of them. </p>
                         </div>
 
                         <div class="sect-5">
-                          <p>Playing the guitar.</p>
+                          <p style = "font-size: 14pt">Playing the guitar.</p>
                           <p><button onclick="addRandomSong()">My Favorite Songs to Play</button>
                           <div id="song-container"></div></p>
-                    
-                          <p>Musicals.</p>
-                          <p><button onclick="addRandomMusical()">My Favorite Musicals</button>
-                          <div id="musical-container"></div></p>
                         </div>
 
                         <div class="sect-6">
-                          <p>Making art.</p>
-                          <p><button onclick="document.location = 'artimages.html'">
-                                     Some of my Art</button></p>
-                          <p>Hiking, exploring, looking at, and traveling the world. </p>
-                          <p><button onclick="document.location = 'images.html'">
-                                     My Happy Places</button></p>
+                          <p style = "font-size: 14pt">Creating art.</p>
+                          <p><button onclick = "document.location = 'artimages.html'">Check it out</button>
+                        </div>
+
+                        <div class="sect-7">
```

Comment:
you may want to give your sections more meaningful names. particularly since you have so many of them now

Reply:
Fixed this in next commit